third_party/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -732,10 +752,11 @@ fn compile_source_unit_v2(
     let unit = if units.len() != 1 {
         anyhow::bail!("expected either one script or one module")
     } else {
-        match units.pop().unwrap() {
-            AnnotatedCompiledUnit::Module(m) => (Some(m.named_module.module), None),
-            AnnotatedCompiledUnit::Script(s) => (None, Some(s.named_script.script)),
-        }
+        units.pop().unwrap()
+        // match units.pop().unwrap() {
```

Comment:
remove commented code

Reply:
@brmataptos The commented code has not been removed yet.

Reply:
Done now.